config/locales/simple_form.fa.yml

```diff
@@ -4,14 +4,20 @@ fa:
     hints:
       defaults:
         avatar: یکی از قالب‌های PNG یا  GIF یا JPG. بیشترین اندازه ۲ مگابایت. تصویر به اندازهٔ ۱۲۰×۱۲۰ پیکسل تبدیل خواهد شد.
-        display_name: '%{count} حرف باقی مانده'
+        display_name:
+            one: '<span class="name-counter">1</span> حرف باقی مانده'
```

Comment:
Oh I now get it. Thank you for bearing with me! Weird. I just copied it from `simple_form.en.yml`.